src/Microsoft.DotNet.XHarness.Tests.Runners/Microsoft.DotNet.XHarness.Tests.Runners.csproj

```diff
@@ -10,6 +10,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" Version="5.0.0-beta.20254.1" />
     <PackageReference Include="Mono.Options" Version="6.6.0.161" />
+    <PackageReference Include="NUnit" Version="3.12.0" />

```

Comment:
I started thinking about this recently, I believe that we will have to split this. The main issue, we have xunit and nunit, which add their own dependencies, but even worse, we will have the wasm runner.. so it might be now the time to split it.

@MattGal comments? 

It can be done in this PR

Reply:
Yes please; NUnit is 3.3 MB and runtime I believe will never use it, so keeping that out of builds and the runner makes a lot of sense.